packages/optimistic-oracle/src/proposer.js

```diff
@@ -73,60 +73,209 @@ class OptimisticOracleProposer {
     });
 
     for (let priceRequest of this.optimisticOracleClient.getUnproposedPriceRequests()) {
-      const priceFeed = await this._createOrGetCachedPriceFeed(priceRequest.identifier);
+      await this._sendProposal(priceRequest);
```

Comment:
Maybe add a TODO here where we can optionally configure a reward threshold here so the bot only proposes for requests that are adequately rewarded?

This isn't a trivial feature, but I think it's worth noting that this is probably required to keep people from spamming the bot.

Reply:
Greet feature. Will definitely todo for this pr and implement in next pr

Reply:
Because we need to translate reward amounts between currencies, this may be something we save for a v2 since it may be pretty complicated to implement... but definitely interested to hear if you have a clever solution to solve this in a simpler way.

Reply:
Yeah simple solution is no need to transalate reward amounts, just denominate the reward threshold in the collateral currency.